server/block/cube/axis.go

```diff
@@ -35,7 +35,7 @@ func (a Axis) RotateLeft() Axis {
 
 // RotateRight rotates an Axis from X to Z or from Z to X.
 func (a Axis) RotateRight() Axis {
-	// No difference in rotating left or right for an Axis.
+	// No difference in rotating left or right for an Facing.
```

Comment:
Is this change right? And should probably clarify that this is referring to the Y axis

Reply:
No, probably a messy refactor.